src/pivot.coffee

```diff
@@ -421,10 +421,13 @@ callWithJQuery ($) ->
         #`order`: Array of "+" or "-" values, one per attribute. "-"
         #  indicates a descending sort.
         arrSort: (attrs, order) =>
+            # Convert empty value string back to null so it may be compared naturally to other values.
+            _convertBlankToNull = (attrVal) => return if attrVal == @emptyValue then null else attrVal
+
             sortersArr = (getSort(@sorters, a) for a in attrs)
             (keyA,keyB) ->
                 for own attrIdx, sorter of sortersArr
-                    comparison = sorter(keyA[attrIdx], keyB[attrIdx])
+                    comparison = sorter(_convertBlankToNull(keyA[attrIdx]), _convertBlankToNull(keyB[attrIdx]))
```

Comment:
Naming issue:  Might want to avoid assuming "empty" is defined as "blank" since this is configurable.